core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala

```diff
@@ -149,6 +150,12 @@ private[ui] class ExecutorTable(stageId: Int, stageAttemptId: Int, parent: Stage
                 {Utils.bytesToString(v.diskBytesSpilled)}
               </td>
             }}
+            <td>
+              {val logs = parent.executorsListener.executorToLogUrls.getOrElse(k, Map.empty)
+               logs.map {
+                case (logName, logUrl) => <div><a href={logUrl}>{logName}</a></div>
+              }}
```

Comment:
I'll admit, I'm still extremely new to Scala and so I haven't really picked up a good sense of formatting yet and tried to follow a convention similar to lines above this.  Would this https://gist.github.com/tmagrino/1dddaa4f2a9b0961d98809482d0cc4bb be considered better?